tests/test_datasets.py

```diff
@@ -66,10 +82,20 @@ def test_dataset_item_shapes(datastore_name):
     assert forcing.ndim == 3
     assert forcing.shape[0] == N_pred_steps
     assert forcing.shape[1] == N_gridpoints
-    assert forcing.shape[2] == datastore.get_num_data_vars("forcing") * (
+    # each time step in the window has one corresponding temporal embedding
```

Comment:
Agree that this should not be referred to as an embedding (this is rather what we will use to create an embedding later). I've also been sloppily calling it this :sweat_smile: I think something like time delta is more clear (forcing increments makes me think it is the increment in forcing values, rather than in time).